examples/material-ui-nextjs-ts/src/app/layout.tsx

```diff
@@ -47,7 +47,7 @@ export default function RootLayout({ children }: { children: React.ReactNode })
           <AppBar position="fixed" sx={{ zIndex: 2000 }}>
             <Toolbar sx={{ backgroundColor: 'background.paper' }}>
               <DashboardIcon sx={{ color: '#444', mr: 2, transform: 'translateY(-2px)' }} />
-              <Typography variant="h6" noWrap component="div" color="black">
+              <Typography variant="h6" noWrap component="div">
```

Comment:
Sort off-topic to the PR but I'm curious about why render this typography element as `div` 🤔

Reply:
Great question! It doesn't seem to have any effect when removed so I went ahead and dropped it. I also realized that the text was rendering as white for some reason when the color wasn't defined in light mode, which I assume is why the original author hardcoded it black. So I just added `color="text.primary"` to make it happy in all modes.

Reply:
OK I think I know why it was a div before! I just learned that the `noWrap` prop only works on block and inline-block elements, so the original author must've wanted to make this a block so it would work. Personally, I don't think `noWrap` is necessary at all here so I dropped it, too.

Reply:
Thanks for circling back — it makes a ton of sense!